src/Microsoft.ML.StandardLearners/Standard/doc.xml

```diff
@@ -21,10 +21,54 @@
           Elastic net regularization can be specified by the 'L2Const' and 'L1Threshold' parameters. Note that the 'L2Const' has an effect on the rate of convergence.
           In general, the larger the 'L2Const', the faster SDCA converges.
         </para>
-        <a href='https://www.microsoft.com/en-us/research/wp-content/uploads/2016/06/main-3.pdf'>Scaling Up Stochastic Dual Coordinate Ascent</a>.
-        <a href='http://www.jmlr.org/papers/volume14/shalev-shwartz13a/shalev-shwartz13a.pdf'>Stochastic Dual Coordinate Ascent Methods for Regularized Loss Minimization</a>.
-      </remarks>
+        <para>For more information, see also:</para>
+        <list>
+          <item><description>
+            <a href='https://www.microsoft.com/en-us/research/wp-content/uploads/2016/06/main-3.pdf'>Scaling Up Stochastic Dual Coordinate Ascent</a>.
+          </description></item>
+          <item><description>
+            <a href='http://www.jmlr.org/papers/volume14/shalev-shwartz13a/shalev-shwartz13a.pdf'>Stochastic Dual Coordinate Ascent Methods for Regularized Loss Minimization</a>.
+          </description></item>
+        </list>
+       </remarks>
     </member>
+    <example name="StochasticDualCoordinateAscentBinaryClassifier">
+      <example>
+        <code language="csharp">
+          new StochasticDualCoordinateAscentBinaryClassifier()
```

Comment:
>() [](start = 60, length = 2)

let's reconcile the usage of `()` across examples?

Reply:
I like it better without the parentheses

---
In reply to: [203198608](https://github.com/dotnet/machinelearning/pull/529#discussion_r203198608) [](ancestors = 203198608)